.github/workflows/lint.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: "Lint"
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@master
+        with:
+          node-version: 14.4.0
```

Comment:
Let's use the node version specified in the .nvmrc file at root. Is it possible to instead just use `nvm` in this job script, and run `nvm install` & `nvm use`? This way we don't write the version number in multiple places.

Reply:
While, we could, setup-node action is probably better as it caches node version so it's not re-downloaded on every action run. I think I can read node-version variable from .nvmrc file anyways.